### ISSUE TITLE
scanning enhancements

### ISSUE THREAD

**Issue by andrewrk** [andrewrk/groovebasin#458]
* Don't scan albums (See #194)
 * Additionally compute audio checksum (maybe as simple as crc32). This will be used when updating tags to ensure that audio data was not corrupted.
 * Additionally compute song waveform. I saw a picture of @moonwave99's [player](https://github.com/moonwave99/playa) and it looks great. This will also be beneficial if we decide to add ability to ignore sections of tracks (such as long sections of silence in the beginning, middle, or end)

Consider adding these features directly to libgroove.

**Comment by andrewrk** [comment]
@moonwave99 check it out, I copied your design because it looks great :-)

![](http://i.imgur.com/PAH94EP.png)

![](http://i.imgur.com/zCoUO4x.png)

**Comment by moonwave99** [comment]
Oh, I am delighted ^^ Hope to go back on track w/ my player soon, so I can start working on libgroove packaging.

Great job!

**Comment by andrewrk** [comment]
You might consider waiting until libgroove 5.0.0 is released. I'm breaking compatibility in order to make some core improvements, such as swapping out SDL for [libsoundio](http://libsound.io/).  I'm carefully documenting what you have to do in order to upgrade though so I'm hoping it won't be too painful.

**Comment by moonwave99** [comment]
Yeah I am following the updates there ^^ as long as `node-groove` is updated accordingly, I should be fine!
